test/jdk/javax/rmi/ssl/SSLSocketParametersTest.java

```diff
@@ -21,16 +21,30 @@
  * questions.
  */
 
+/*
+ * @test
+ * @bug 5016500
+ * @summary Test SslRmi[Client|Server]SocketFactory SSL socket parameters.
+ * @run main/othervm SSLSocketParametersTest 1
+ * @run main/othervm SSLSocketParametersTest 2
+ * @run main/othervm SSLSocketParametersTest 3
+ * @run main/othervm SSLSocketParametersTest 4
+ * @run main/othervm SSLSocketParametersTest 5
+ * @run main/othervm SSLSocketParametersTest 6
+ * @run main/othervm SSLSocketParametersTest 7
```

Comment:
Is a fresh VM needed for each of these tests?
From a naive point of view it appears that 1, 2, 3, 4, 6, 7 use the same system properties.
The tests would complete more quickly if they could run the compatible tests in a single VM.
Perhaps main() could iterate over the args[n] and run the corresponding test.

Reply:
I tried this suggestion but test case 6 failed. From the logs, it looked like an object was being re-used and an exception wasn't being thrown when it should have been.